python/altrios/demos/speed_limit_train_sim_demo.py

```diff
@@ -151,9 +149,62 @@
 ax[-1].set_ylabel('Speed [m/s]')
 ax[-1].legend()
 plt.suptitle("Speed Limit Train Sim Demo")
+
+fig1, ax1 = plt.subplots(3, 1, sharex=True)
```

Comment:
<img width="631" alt="image" src="https://github.com/NREL/altrios/assets/854536/421c52fd-48d7-4935-bbdd-b0175adca26c">

@SWRIganderson , I'd sure appreciate some serious scrutiny on this plot and similar plots in other contexts